js/models/listing/Listing.js

```diff
@@ -253,6 +256,8 @@ export default class extends BaseModel {
     const parsedResponse = response.listing;
 
     if (parsedResponse) {
+      // set the hash
+      parsedResponse.listingHash = response.hash;
```

Comment:
ok, it's not such a huge deal, leave it be.

But for consistency can we preface all the other listing attributes with listing as well (listingCategory, listingSlug, etc...)? ;)